woocommerce/wc-admin
# Deprecate the no longer used Count component

Count is no longer used by us and the styles look out of date. Recently we introduced Badge which fulfills the same role so I'm opting to deprecate this as soon as possible so we can remove in the next major version of `@woocommerce/components`

## Testing instructions

1. Introduce a `Count` component to the wc-admin dashboard somewhere
2. Build the app as usual ie `npm i && npm run clean && npm start`
3. Run the app, navigate to dashboard, check the console and confirm a deprecation warning is printed

No changelog required (I've provided one in the package)
